api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -275,6 +275,31 @@ message ExternalProcessor {
   // backend stream lifetime. In this case, Envoy will eventually timeout the external processor stream according to this time limit.
   // The default value is 5000 milliseconds (5 seconds) if not specified.
   google.protobuf.Duration deferred_close_timeout = 19;
+
+  // [#not-implemented-hide:]
+  // Enables sending body to the side stream server before receives the header response from that server.
+  // It only works for STREAMED body processing mode. For any other body processing modes, it is ignored.
+  //
+  // Setting this flag to true is required in cases where the side stream server must wait for at least one
+  // chunk of body before sending the header response.
+  //
+  // The side stream server must not send a separate header response in this case. Instead, it encodes the
+  // response headers in the first body response and sends it back to Envoy.
```

Comment:
if this restriction is required, shouldnt the protocol be updated indicating that this bit is set and envoy is expecting header response as part of body response?

Reply:
That's a good question, maybe we should send this configuration to the side stream server in the ProcessRequest for header. Let me make the API change.